src/main/kotlin/pet/project/app/controller/UserController.kt

```diff
@@ -0,0 +1,42 @@
+package pet.project.app.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import pet.project.app.dto.user.RequestUserDto
+import pet.project.app.dto.user.ResponseUserDto
+import pet.project.app.dto.user.UserMapper.toDto
+import pet.project.app.dto.user.UserMapper.toModel
+import pet.project.app.service.UserService
+
+@RestController
+@RequestMapping("/user")
+class UserController(private val userService: UserService) {
+    @PostMapping("/")
+    @ResponseStatus(HttpStatus.CREATED)
+    fun create(@RequestBody userDto: RequestUserDto): ResponseUserDto {
+        val createdUser = userService.create(userDto.toModel())
+        return createdUser.toDto()
+    }
+
+    @GetMapping("/{id}")
+    fun getById(@PathVariable("id") userId: String): ResponseUserDto {
+        return userService.getById(userId).toDto()
+    }
```

Comment:
[nit] in Kotlin you can use shorter syntax
https://kotlinlang.org/docs/functions.html#single-expression-functions
```suggestion
    fun getById(@PathVariable("id") userId: String): ResponseUserDto =
       userService.getById(userId).toDto()
```

Reply:
Fixed: Investigated code conventions and updated as suggested